apps/src/maze/maze.js

```diff
@@ -631,12 +597,11 @@ module.exports = class Maze {
   /**
    * Animates a single action
    * @param {string} action The action to animate
-   * @param {boolean} spotlightBlocks Whether or not we should highlight entire blocks
    * @param {integer} timePerStep How much time we have allocated before the next step
    */
-  animateAction_(action, spotlightBlocks, timePerStep) {
+  animateAction_(action, timePerStep) {
     if (action.blockId) {
-      studioApp().highlight(String(action.blockId), spotlightBlocks);
+      studioApp().highlight(String(action.blockId));
```

Comment:
are we still using the spotlight option for this method elsewhere, or can it be removed there, too?

Reply:
This is the last place we use it AFAIK!

But I don't want to remove it just yet — I'd like to re-purpose the spotlight UX for selecting multiple blocks ([spec](https://docs.google.com/document/d/1wJMKXJS6Z8tsYQRRq93r7k4TTzG4jQbIXfKdKcSTZTI/edit#heading=h.7w6qf1s2ik7g)).